### PR TITLE
Database Mail Profile Name Configuration for SQL Managed Instance

### PR DESCRIPTION
**Pull request by AndresPad**
We should add in a note in the database configuration instructions for SQL Managed Instances that instruct users to create a profile name 'AzureManagedInstance_dbmail_profile'.  Otherwise, Managed Instance will be unable to send emails via SQL Agent.  
https://techcommunity.microsoft.com/t5/azure-sql/sending-emails-in-azure-sql-managed-instance/ba-p/386235
https://johnmccormack.it/2020/10/how-do-i-set-up-database-mail-for-azure-sql-db-managed-instance/

### CODE REVIEW

File: docs/relational-databases/database-mail/configure-database-mail.md [hunk 1]
```diff
@@ -341,6 +341,9 @@ ms.author: sstein
   
  **Profile Name**  
  Displays the name of the profile.  
+ 
+> [!NOTE]  
+>  If your are configuring database mail for an SQL Managed Instance, and you want to send messages using SQL Agent jobs, you will need a profile named '**AzureManagedInstance_dbmail_profile**', otherwise Managed Instance will be unable to send emails via SQL Agents. [Sending emails in Azure SQL Managed Instances](https://techcommunity.microsoft.com/t5/azure-sql/sending-emails-in-azure-sql-managed-instance/ba-p/386235).
```
Comment (WilliamDAssafMSFT) [inline docs/relational-databases/database-mail/configure-database-mail.md:346]:
```suggestion
>  If you are configuring database mail for an Azure SQL Managed Instance, and you want to send messages using SQL Agent jobs, you must create a database mail profile named '**AzureManagedInstance_dbmail_profile**'. For more information and a sample script, see [SQL Managed Instance SQL Agent job notifiations](/azure/azure-sql/managed-instance/job-automation-managed-instance).
```

**Comment by PRMerger14** [comment]
@WilliamDAssafMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/database-mail/configure-database-mail.md [hunk 2]
```diff
@@ -341,6 +341,9 @@ ms.author: sstein
   
  **Profile Name**  
  Displays the name of the profile.  
+ 
+> [!NOTE]  
+>  If you are configuring database mail for an Azure SQL Managed Instance, and you want to send messages using SQL Agent jobs, you must create a database mail profile named '**AzureManagedInstance_dbmail_profile**'. For more information and a sample script, see [SQL Managed Instance SQL Agent job notifiations](/azure/azure-sql/managed-instance/job-automation-managed-instance).
```
Comment (WilliamDAssafMSFT) [inline docs/relational-databases/database-mail/configure-database-mail.md:346]:
```suggestion
>  If you are configuring database mail for an Azure SQL Managed Instance, and you want to send messages using SQL Agent jobs, you must create a database mail profile named '**AzureManagedInstance_dbmail_profile**'. For more information and a sample script, see [SQL Managed Instance SQL Agent job notifications](/azure/azure-sql/managed-instance/job-automation-managed-instance#sql-agent-job-notifications).

```

**Comment by WilliamDAssafMSFT** [comment]
Thanks @AndresPad for the contribution! This is documented in a couple places but not here in the SQL Server side, a good add. I've added a different link to the proper documentation in the Azure docs repo. 

Your commit will be merged and this article update is in flight. After the updated article is published, GitHub will list you as an article contributor. 

Thanks again for this feedback, your suggestions help improve our documentation.

#sign-off

**Comment by PRMerger12** [comment]
@WilliamDAssafMSFT: I'm sorry - only the author of this article, @stevestein, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by PRMerger20** [comment]
@WilliamDAssafMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**